autogen.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
+
 echo "Run autoreconf instead."
 echo "Example: "
 echo "	$ autoreconf -f -i -v"
+
+exit 64
```

Comment:
From sysexits(3):

```
 EX_USAGE (64)	   The command was used incorrectly, e.g., with the
			   wrong number of arguments, a bad flag, a bad syntax
			   in a parameter, or whatever.
```

It's the erroneous return value for when instructions are shown.

I ran into issues with autogen.sh returning true (as it did before), because I instinctively ran it, && ./configure && make, and my shell kept running commands as if it had succeeded.